src/Windows/dll/dcap_provider.def

```diff
@@ -1,12 +1,10 @@
-LIBRARY dcap_quoteprov
-EXPORTS
-    sgx_ql_get_quote_config
-    sgx_ql_free_quote_config
-    sgx_ql_get_revocation_info
-    sgx_ql_free_revocation_info
-    sgx_get_qe_identity_info
-    sgx_free_qe_identity_info
```

Comment:
Removing sgx_get/free_qe_identity_info will force users to have to use the latest OE sdk.  I suggest keeping this at least for one release to make it easier for users to upgrade one library at a time?

Reply:
I've brought these back in - will remove in next release. Since we aren't testing these anymore, I really don't like keeping them around. When was this removed from OE SDK?

Reply:
The latest one v0.9, so I think in your next release you should be able to remove it.